src/color-modes/themes/dark.scss

```diff
@@ -1,6 +1 @@
-@import '../../support/index.scss';
-@import '@primer/primitives/dist/scss/colors/_dark.scss';
-
-@include color-mode-theme(dark) {
-  @include primer-colors-dark;
-}
+@import '@primer/primitives/dist/css/functional/themes/dark.css';
```

Comment:
If we're gonna go with a major, I say we just completely remove the `src/color-modes/` folder and use the primitives imports directly in dotcom

Reply:
🔥